gamedata/scripts/mags_patches.script

```diff
@@ -318,27 +319,59 @@ local string_gsub   = string.gsub
 
 CloneWep = _G.alife_clone_weapon
 function _G.alife_clone_weapon(se_obj, section, parent_id)
+print_dbg("clone weapon")
```

Comment:
if we confirm this works, can we remove these prints

Reply:
I ment  to get that one when i got the others. I'm leaving the data copy print for now. I added IDs to it.